index.js

```diff
@@ -254,11 +254,7 @@ function processFormats(formats) {
   const filteredFormats = formats.filter((format) => filterFormatCodecs(format) && filterFormatFps(format));
   filteredFormats.sort(videoTrackSort)
 
-  const selected4K = filteredFormats.find((format) => (format.height === 2160 && format.acodec !== 'none')) || filteredFormats.find((format) => format.height === 2160);
-  const selected1080p = filteredFormats.find((format) => (format.height === 1080 && format.acodec !== 'none')) || filteredFormats.find((format) => format.height === 1080);
-  const selectedLowQuality = filteredFormats.find((format) => (format.height === 720 && format.acodec !== 'none')) || filteredFormats.find((format) => format.height === 720) || filteredFormats.find((format) => (format.height <= 720 && format.acodec !== 'none')) || filteredFormats.find((format) => (format.height <= 720));
```

Comment:
We do filtering on the box and try to pick out the combined one if `allowSplitTracks` is not true so this kinda pointless.

Reply:
Resolving this conversation as its no longer relevant